zx.mjs

```diff
@@ -30,6 +31,7 @@ Object.assign(global, {
   chalk,
   fs,
   os,
+  require: createRequire(import.meta.url)
```

Comment:
So require will be relative of zx bin? 

Maybe make it relative to the executed script?

Reply:
1) I _believe_  there was a reason why `createRequire` was added. I have not seen any plans to remove this API.
2) Reasonable.